#!/usr/bin/env python
PACKAGE = "perception"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("modified_mask_path", str_t, 0,               "modified mask path", "~/somefolder/anotherfolder/filename.pbm")

exit(gen.generate(PACKAGE, "perception", "PreprocessingPixelwiseMask"))
